pages/index.js

```diff
@@ -1,3 +1,23 @@
+import styled from "styled-components";
+import FoodCard from "../components/FoodCard";
+import useSWR from "swr";
+import { fetcher } from "../helpers/api";
+
 export default function Home() {
-  return <h1>Hello Flamingos! 🦩</h1>;
+  const { data: recipes, error } = useSWR("/api/recipes", fetcher);
+
+  if (error) return <h1>There was an error</h1>;
+
```

Comment:
You could add the case for when recipes is empty or otherwise unavailable, and return some JSX that says something like "data is loading ...".